packages/plans-grid-next/src/style.scss

```diff
@@ -701,6 +655,41 @@ $mobile-card-max-width: 440px;
 			color: var(--studio-orange-40);
 		}
 	}
+
+	.components-custom-select-control {
```

Comment:
Thanks for catching this and scoping the styling. I smoke tested the storage add-on dropdown in `/start/plans` and `/plans`, and behavior looks to be as expected on desktop and mobile 👍